src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -269,6 +269,17 @@ class ApplicationManagerImplTest
     ASSERT_TRUE(mock_app_ptr_.get());
   }
 
+  void SetUpCreateModuleInfoSOExpectation(
```

Comment:
@ydementieiev @mvorobio `SetExpectationForCreateModuleInfoSO` ?

Reply:
@AByzhynar  thank you, [renamed](https://github.com/LuxoftSDL/sdl_core/pull/38/commits/0a5924e2dc93b929033b4e10a282660445f7580e)